packaged_helloworld/setup.py

```diff
@@ -0,0 +1,13 @@
+import packaged_helloworld
+from setuptools import find_packages, setup
+
+package_info = {
+    "name": "packaged_helloworld",
+    "version": packaged_helloworld.__version__,
+    "entry_points": {
+        "zulip_bots.registry": ["packaged_helloworld=packaged_helloworld.packaged_helloworld"],
+    },
+    "packages": find_packages(),
+}
+
+setup(**package_info)
```

Comment:
Regardless, we need to improve our automated release process if we are going to do this. :)

Reply:
I think it won't be necessary. Since this package is only for demonstration and `zulip_bots` hardly rely on it.

Reply:
Yeah, I think it's probably fine to have this be in the same repository for now.  When we write documentation for this, I think what we'll actually want is a `zulip-new-bot bot_name` command that creates a directory of this form for you, so that we can have really simple steps for making a new bot.